src/filerepository.js

```diff
@@ -265,12 +283,13 @@ class FileLoader {
 		this.id = uid();
 
 		/**
-		 * A `File` instance associated with this file loader.
+		 * Additional wrapper over a file promise passed to this loader.
 		 *
-		 * @readonly
-		 * @member {File}
+		 * @see #_createFilePromiseWrapper
+		 * @private
+		 * @member {Object} wrapper
```

Comment:
Might be nice to create `@typedef` for the wrapper to document it instead of just `Object`.